mftf/2.3/metadata.md

```diff
@@ -70,11 +73,13 @@ The following diagram demonstrates the XML structure of a metadata file:
 
 ## Principles   {#principles}
 
-* The `dataType` value must match the `type` value of the corresponding entity.
-* The file name contains data type split with `_` and ends with `-meta`.
-Example: `product_attribute-meta.xml`.
-* A metadata file may contain different types of operations (`type`) with the same data entity (`dataType`).
-Example: 
+1. The `dataType` value must match the `type` value of the corresponding entity.
+2. The file name contains data type split with `_` and ends with `-meta`.
+   Example: `product_attribute-meta.xml`.
```

Comment:
file name can be anything but needs to ends with `-meta`. We use data type a lot by convention.